lib/defra_ruby/exporters/registration_export_service.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require "csv"
+require_relative "registration_epr_export_report"
+
+module DefraRuby
+  module Exporters
+    class RegistrationExportService
+
+      EPR_EXPORT_FILENAME = "waste_exemptions_epr_daily_full.csv"
+
+      def epr_export
+        file_path = full_path
+        write_to_file(RegistrationEprExportReport, file_path)
+        save_to_s3(:epr, file_path)
```

Comment:
There is no delete after we have uploaded to AWS S3, but I think it would be prudent to do so rather than leaving the file on our instances.

Could we delete the locally generated file as the final step in the export process?

Reply:
Fixed in cab4157.